client/styles/styles.css

```diff
@@ -355,7 +355,4 @@ img {
 :-ms-input-placeholder {
    color: white;
 }
-.blurb {
-  background-color: black;
-  color: white;
-}
\ No newline at end of file
+=======
```

Comment:
Viewed this in context and I'm guessing it's leftover formatting from maybe a merge conflict.  Merging now and I'll clean it up. @justin-lai